hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/hdds/scm/pipeline/TestPipelineManagerImpl.java

```diff
@@ -628,6 +635,31 @@ public void testScrubPipelines() throws Exception {
     pipelineManager.close();
   }
 
+  @Test
+  public void testScrubOpenWithUnregisteredNodes() throws Exception {
+    PipelineManagerImpl pipelineManager = createPipelineManager(true);
+    pipelineManager.setScmContext(scmContext);
```

Comment:
nit. I see other tests doing this as well but I'm not sure it's necessary since `createPipelineManager(boolean)` is passing this variable to the `PipelineManagerImpl` constructor.

Reply:
You are correct. I just copied the new test from an old one and adapted. I have removed that line from all the tests and they all pass locally.

Reply:
At that time with Junit4, I had some issues with the test running, So I added it. I just checked now it is safe to remove. Thanks for cleaning it up @sodonnel.